vision/module/location.py

```diff
@@ -253,10 +258,10 @@ def _circleDetection(self) -> np.ndarray:
         gray = cv2.cvtColor(src=self.img, code=cv2.COLOR_RGB2GRAY)
 
         # Guassian Blur
```

Comment:
Comment needs to change to reflect updated blur. Documentation here fixed by Mateusz, so just needs to be fixed in the merge conflict.